apps.yaml

```diff
@@ -1,32 +1,13 @@
-aiidalab-qe:
-  categories:
-  - technology-aiida
-  - technology-aiidalab
-  - quantum
-  git_url: https://github.com/aiidalab/aiidalab-qe.git
+my-big-map-app:
   metadata:
-    authors:
-      $ref: https://aiidalab.github.io/aiidalab-qe/metadata.json#authors
+    title: DFT_VASP
     description: |
-      Compute band structures and other structure properties with Quantum ESPRESSO
-      on the AiiDAlab platform.
-    logo: https://raw.githubusercontent.com/aiidalab/aiidalab-qe/master/miscellaneous/logos/QE.jpg
-    state:
-      $ref: https://aiidalab.github.io/aiidalab-qe/metadata.json#state
-    title: Quantum ESPRESSO AiiDAlab app
-    external_url: https://www.materialscloud.org/work/aiidalab
-    version:
-      $ref: https://aiidalab.github.io/aiidalab-qe/metadata.json#version
-optimade-web:
-  categories:
-  - technology-materials-cloud
-  - utilities
-  metadata:
-    authors: Casper Welzel Andersen
-    description: |
-      Graphical client hosted on Materials Cloud to search databases that
-      implement the OPTIMADE API (https://www.optimade.org/).
-    external_url: https://www.materialscloud.org/work/tools/optimadeclient
-    logo: https://www.materialscloud.org/mcloud/api/v2/work/tools/logos/optimade.png
```

Comment:
Please just add the new app at the bottom, but do not to delete existing entries.